packages/sdk-demo/src/apps/SDKDemo/SDKDemo.tsx

```diff
@@ -136,3 +171,44 @@ const MoniteReactQueryDevtools = () => {
   const { queryClient } = useMoniteContext();
   return <ReactQueryDevtools initialIsOpen={false} client={queryClient} />;
 };
+
+const AuthErrorsBackdrop = ({
+  errors,
+  logout,
+}: {
+  errors: Array<Error>;
+  logout: () => void;
+}) => {
+  if (!errors.length) return null;
+
+  return (
+    <Backdrop open>
+      <Stack
+        spacing={2}
+        maxWidth="sm"
+        margin="auto"
+        height="100vh"
+        justifyContent="center"
+      >
+        {errors.map((error) => (
+          <Alert severity="error" key={error.message}>
+            Error: <code>{error.message}</code>
+          </Alert>
+        ))}
+        {Boolean(errors.length) && (
```

Comment:
The `Boolean(errors.length)` check is redundant here, due to the initial check before `if (!errors.length) return null;`. You can safely remove this conditional rendering

Reply:
Thanks!